deps/Makefile

```diff
@@ -81,3 +83,27 @@ jemalloc: .make-prerequisites
 	cd jemalloc && $(MAKE) CFLAGS="$(JEMALLOC_CFLAGS)" LDFLAGS="$(JEMALLOC_LDFLAGS)" lib/libjemalloc.a
 
 .PHONY: jemalloc
+
+ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+SSL_CONFIG= ./config -fPIC no-shared  --prefix=$(ROOT_DIR)/s2n/libcrypto-root/
+
+ifeq ($(uname_S),Darwin)
+	# Make isinf() available
+	SSL_CONFIG= ./Configure --prefix=$(ROOT_DIR)/s2n/libcrypto-root darwin64-x86_64-cc
+endif
+
+
+openssl: .make-prerequisites
+	@printf '%b %b\n' $(MAKECOLOR)MAKE$(ENDCOLOR) $(BINCOLOR)$@$(ENDCOLOR)
+	cd openssl && $(SSL_CONFIG)
+	cd openssl && $(MAKE) depend
+	cd openssl && $(MAKE)
+	cd openssl && $(MAKE) install
```

Comment:
This can overwrite / blow away system libraries and docs. May want to remove this line.